addons/udes_sale_stock/data/ir_cron.xml

```diff
@@ -16,5 +16,21 @@
             <field name="code">model.confirm_orders()</field>
         </record>
 
+        <!-- Confirm Sale Orders and Reserve Stock in a single cron job -->
+        <record id="merge_confirm_orders_reserve_stock" model="ir.cron">
+          <field name="name">Merge Confirm Orders and Reserve Stock</field>
+            <field name="active" eval="False" />
+            <field name="user_id" ref="base.user_root" />
+            <field name="interval_number">5</field>
+            <field name="interval_type">minutes</field>
+            <field name="numbercall">-1</field>
+            <field name="doall">0</field>
+            <field name="model_id" ref="udes_sale_stock.model_sale_order"/>
+            <field name="state">code</field>
+            <field name="code">
+                model.merge_confirm_orders_reserve_stock(confirm_orders_batch_size=1000, confirm_orders_commit_size=500, finish=False)
```

Comment:
Thought to make reduce the commit size by default because after that is reserve_stock cron job too. There is not any very clear idea why, but can be changed through UI. I put in the cron method to be easier to change it from UI so we know what are the optional variable names